generators/v1/acr/acr.go

```diff
@@ -104,6 +105,24 @@ func (g *Generator) Cleanup(_ context.Context, _ *apiextensions.JSON, _ genv1alp
 	return nil
 }
 
+// GetKeys returns the keys for the generator.
+func (g *Generator) GetKeys() map[string]string {
+	return map[string]string{
+		"username": "Default login username for Azure Container Registry (ACR)",
```

Comment:
IIUC, these are the fields in Spec object, and short description of those fields. Should we have a generator for this too? I think we can use `go/ast` package to parse the comments of these fields. It's over engineering, but I think it will help maintenance.